### DISCUSSION TITLE
"Add" button after repo search not working

### DISCUSSION THREAD

**Discussion by leoplusx** [paircodedev/sandbox#7] [Bug Report]
I search for a repo. Matches are displayed correctly. I click the "Add" button for one of them. But nothing happens. Also, on the next page, I can see that no repos have been saved.

**Comment by Necmttn** [comment]
Interesting; I tried replicating it but was unable to. Please let me know if the problem occurs again.

You are the best @leobaumgardt, thank you so much for reporting!

**Reply by leoplusx** [reply]
Weird. Am using Chrome on Mac (Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/90.0.4430.212 Safari/537.36).

Had the error each time I returned to that page (and I tried several times).

I will try to get more data if I post another job and the problem occurs again (video, console output, things like that).
